### ISSUE TITLE
[text-to-speech] Add part_of_speech parameter for customization words (Japanese only)

### ISSUE THREAD

**Issue by jeffpk62** [watson-developer-cloud/node-sdk#376]
This is mostly a trivial change the Node SDK for Text to Speech.  The following methods can include a part_of_speech field as part of the request or response related to individual words for Japanese models:

- updateCustomization() (passed as part of request words array object)
- getCustomization() (included in response)
- addWord() (passed as request parameter in params)
- addWords() (passed as part of request words array object)
- getWord() (include in response)
- getWords() (included in response)

You can take a look at https://watson-api-explorer.mybluemix.net/apis/text-to-speech-v1 for specific details of where the field can appear and how it's used.

**Comment by nfriedly** [comment]
I believe no change is needed for the "included in response" methods, the SDK passes that JSON along without modification.

**Comment by jeffpk62** [comment]
Yeah, the only change for the responses would be to correct the examples you show in the JSDoc.  For instance, where the doc says that the response is {word, translation}, it can actually be {word, translation, part_of_speech}.  I thought you might want to note that in the appropriate cases.

**Comment by nfriedly** [comment]
Aah, good point.
